packages/api/src/sentry.ts

```diff
@@ -23,20 +24,30 @@ export function initSentry(app: Application): void {
     sampleRate: 1.0,
     // Traces sample rate
     tracesSampler: samplingContext => {
-      // Do not send health checks to Sentry
-      if (samplingContext.transactionContext.name === "GET /") {
-        return 0;
-      }
-      // Sample 1% of OPTIONS to have some visibility
-      if (samplingContext.transactionContext.name.match(/^OPTIONS.*$/)) {
+      if (isTracingDisabledForTx(samplingContext.transactionContext)) {
         return 0;
       }
       // TODO #499 Review this based on the load on our app and Sentry's quotas
-      return 0.3;
+      return 0.5;
```

Comment:
Since we removing a lot of requests, I'm thought we could try to increase to 50% and see when we hit the quota.